test/fixtures/definitions.txt

```diff
@@ -83,5 +153,176 @@ text with **inlines**, \[backslash escapes\], typography(tm), "smart quotes", ch
 .
 <p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
-<blockquote>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</blockquote>
+<blockquote>
+<p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
+</blockquote>
+.
+
+Thematic breaks and headings should be rendered within indented quotes
+.
+    * * *
+
+    ---
+
+    # ATX Heading
+
+    SEText Heading
+    --------------
+.
+<blockquote><hr /></blockquote>
+<blockquote><hr /></blockquote>
+<blockquote><h1>ATX Heading</h1></blockquote>
+<blockquote><h2>SEText Heading</h2></blockquote>
+.
+
+Indented quotes separated by blank indented lines (spaces or tabs) are part of the same block quote.
+.
+    chunk1
+    
+      chunk2
+.
+<blockquote>
+<p>chunk1</p>
+<p>chunk2</p>
+</blockquote>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when not separated from the previous paragraph.
+.
+Foo
+    bar
+.
+<p>Foo
+<q class="blockquote">bar</q></p>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when not separated from the subsequent paragraph.
+.
+    foo
+bar
+.
+<p><q class="blockquote">foo</q>
+bar</p>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when they are inside a paragraph with no separation.
+.
+Paragraph text:
+    Indented quote.
+more paragraph text.
 .
+<p>Paragraph text:
+<q class="blockquote">Indented quote.</q>
+more paragraph text.</p>
+.
+
+Blank indented lines around a quote within a paragraph are not rendered.
+.
+Paragraph text:
+    
+    Indented quote.
+    
+more paragraph text.
+.
+<p>Paragraph text:
+<q class="blockquote">Indented quote.</q>
+more paragraph text.</p>
+.
+
+Multiple blockquote paragraphs can be separated by blank indented lines
+.
+Paragraph text:
+    
+    Indented quote paragraph.
+    
+    Second indented quote paragraph.
+    
+more paragraph text.
+.
+<p>Paragraph text:
+<q class="blockquote">Indented quote paragraph.</q>
+<q class="blockquote">Second indented quote paragraph.</q>
+more paragraph text.</p>
+.
+
+Blank lines around indented quotes are not rendered.
+.
+
+    
+    foo
+    
+
+.
+<blockquote>
+<p>foo</p>
+</blockquote>
+.
+
+Indented code blocks: Commonmark #86 (1557)
+> The first line can be indented more than four spaces:
+If the first line of an indented quote is indented 8 spaces, should the entire quote be considered indented?
+.
+        foo
+    bar
+.
+.
+
+Trailing spaces are not rendered in blockquotes.
+.
+    foo  
+.
+<blockquote>
+<p>foo</p>
+</blockquote>
+.
+
+Code fences work within indented quotes.
+.
+    ```
+    aaa
+    ```
+.
+<blockquote><pre><code>
+aaa
+</code></pre></blockquote>
+.
+
+Should link reference definitions work in blockquotes?
```

Comment:
I think perhaps they should, but I believe it would require re-writing the plugin in a different way.